wezterm-client/src/pane/clientpane.rs

```diff
@@ -245,6 +245,12 @@ impl ClientPane {
 
 #[async_trait(?Send)]
 impl Pane for ClientPane {
+    fn get_mouse_cursor_shape(&self) -> Option<String> {
```

Comment:
Take a look at the changes over in https://github.com/wezterm/wezterm/commit/44866cc137e336d8c00b23f5e0d7d1f03983b591#diff-b8703182c50d1b02b59221d8680fa9e6ec9eeb8520beb8e34d85f933fca7fac5R195 for how to do an imperfect but probably good enough in practice implementation of this that will work over the mux protocol